README.rst

```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
 ::
 
 $ python host_os.py build-iso --help
+
+How to contribute a new package to Host OS
+------------------------------------------
+
+Create a new directory at https://github.com/open-power-host-os/versions repository named <package_name>.
+
+Under this new directory: 
+
+1. Create a metadata file <package_name>.yaml. It will contain most of the information required to build the package. The package metadata (yaml) file format is at https://github.com/open-power-host-os/versions/package_metadata_yaml_format.yaml
+2. Create subdirectory "centOS/7", current "Linux Distribution Name"/"Linux Distribution Version", respectively
+3. Create rpm package specification file under <package_name>/centOS/7
+4. Create (copy) the files required to build the package under <package_name>/centOS/7/SOURCES
+ 
+When using %check stage at specification file, add macro to make possible to skip tests (check phase)
```

Comment:
You are using "stage" and "phase" for the same concept, pls be consistent in terminology.

=> If the rpm specification file has a %check macro for running tests during rpm package build, add the following code to allow skipping these tests